index.html

```diff
@@ -24,7 +24,7 @@
     <h1 id="welcome">Welcome!</h1>
     <!-- TODO: Replace this tagline with something less narcissistic -->
     <h3>Striving for student enrichment, technical advancement, and success in the FIRST Robotics Competition.</h3>
-    <a href="/robot/2019" class="button">View our 2019 Robot</a>
+    <a href="/robot/2020" class="button">View our 2020 Robot</a>
```

Comment:
This could similarly automatically be the most recent year

Reply:
True, we'd have to offset the year because we probably only want to show the new robot once the page is ready (normally around late February).